tests/Microsoft.Bot.Builder.Tests/StorageBaseTests.cs

```diff
@@ -11,14 +11,14 @@ namespace Microsoft.Bot.Builder.Tests
 {
     public class StorageBaseTests
     {
-        protected async Task _readUnknownTest(IStorage storage)
+        protected async Task ReadUnknownTest(IStorage storage)
```

Comment:
It's important to keep in mind that, sometimes, renaming a `public` or `protected` method might introduce some breaking changes (as a method could be referenced in several classes). However, I think that in this case won't be any problem, as these methods are referenced in both `CosmosDbStorageTest.cs` (`Bot.Builder.Azure.Test` library) and `MemoryStorageTests.cs` (`Bot.Builder.Test` library) and those references are being updated with this PR. @Bill7zz Do you agree with me?

Reply:
I agree with you, we will move those file into this branch to avoid conflicts that might happen when we rename the methods.